net-libs/tox/tox-9999.ebuild

```diff
@@ -2,18 +2,18 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 inherit autotools eutils git-r3 user systemd
 
 DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
 HOMEPAGE="https://tox.chat"
 SRC_URI=""
-EGIT_REPO_URI="https://github.com/irungentoo/toxcore.git
-	git://github.com/irungentoo/toxcore.git"
+EGIT_REPO_URI="https://github.com/TokTok/c-toxcore.git
+	git://github.com/TokTok/c-toxcore.git"
 
 LICENSE="GPL-3+"
-SLOT="0/0.0"
+SLOT="0/0.1"
```

Comment:
In general sub-slots in live ebuilds are not worth the fight, I would use preserved-libs for this rather. Just a suggestion though.